javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentInstaller.java

```diff
@@ -131,6 +131,16 @@ private static void installBytebuddyAgent(
     // https://bugs.openjdk.org/browse/JDK-8164165
     ThreadLocalRandom.current();
 
+    // AgentBuilder.Default constructor triggers sun.misc.Unsafe::objectFieldOffset called warning
+    // AgentBuilder$Default.<init>
+    // -> NexusAccessor.<clinit>
+    // -> ClassInjector$UsingReflection.<clinit>
+    // -> ClassInjector$UsingUnsafe.<clinit>
+    // -> WARNING: sun.misc.Unsafe::objectFieldOffset called by
+    // net.bytebuddy.dynamic.loading.ClassInjector$UsingUnsafe$Dispatcher$CreationAction
+    // we don't use byte-buddy nexus so we disable it and later restore the original value for the
+    // system property
+    String originalNexusDisabled = System.setProperty("net.bytebuddy.nexus.disabled", "true");
```

Comment:
[minor] Do you think it would be safer to wrap this in a try/finally or even a dedicated method to make it harder to forget about resetting this in the future ?
I think this could potentially impact the application if it relies on bytebuddy as well.

Reply:
moved it to separate method

Reply:
I don't know either, let's try to improve that later if we can.